### PR TITLE
[4.1] adjustments in lang files

### PR DESCRIPTION
**Pull request by heelc29**
### Summary of Changes

corrections in lang files
- remove empty lines
- remove unnecessary spaces
- adjust file header license (in scheduler files)
- alpha sort (caused by #37323)
- correct eol (in com_scheduler.sys.ini)

### Testing Instructions

code review

### Documentation Changes Required

none

### CODE REVIEW

File: administrator/language/en-GB/com_scheduler.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ; Joomla! Project
 ; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
-; GNU General Public License version 2 or later; see LICENSE.txt
+; License GNU General Public License version 2 or later; see LICENSE.txt
```
Comment (infograf768) [inline administrator/language/en-GB/com_scheduler.ini:3]: Using twice the word "License" ?? How do you justify this?

Comment (HLeithner) [reply administrator/language/en-GB/com_scheduler.ini:3]: S/he doesn't need to justiy, all other files uses this format

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on b0166931816fee970192afbf554768c2fc973ad0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37535">issues.joomla.org/tracker/joomla-cms/37535</a>.</sub>

File: administrator/language/en-GB/com_scheduler.ini (hunk 1 above)
Comment (heelc29) [reply administrator/language/en-GB/com_scheduler.ini:3]: > Using twice the word "License" ?? How do you justify this?

Yes, to be consistent with all other files

Comment (infograf768) [reply administrator/language/en-GB/com_scheduler.ini:3]: Indeed. Confused with xml which uses `<license>`. Never remarked before.

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on b0166931816fee970192afbf554768c2fc973ad0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37535">issues.joomla.org/tracker/joomla-cms/37535</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37535">issues.joomla.org/tracker/joomla-cms/37535</a>.</sub>

**Comment by Quy** [comment]
Thank you!

**Pull request merged**